src/ace/pyang/dtnma_amm.py

```diff
@@ -522,6 +531,39 @@ def type_use(parent:str) -> List:
             +type_use('operand')
         ),
     ),
+    Ext('sbr', 'identifier',
+        subs=(
+            OBJ_SUBS_PRE
+            +[
+                ((MODULE_NAME, 'action'), '1'),
+                ((MODULE_NAME, 'condition'), '1'),
+                ((MODULE_NAME, 'min-interval'), '?'), 
+                ((MODULE_NAME, 'max-count'), '?'), 
+                ((MODULE_NAME, 'init-enabled'), '?'), 
+                ('uses', '*'),
+            ]
+        ),
+    ),
+    Ext('tbr', 'identifier',
+        subs=(
+            OBJ_SUBS_PRE
+            +[
+                ((MODULE_NAME, 'action'), '1'),
+                ((MODULE_NAME, 'start'), '?'),
+                ((MODULE_NAME, 'period'), '1'), 
+                ((MODULE_NAME, 'max-count'), '?'), 
+                ((MODULE_NAME, 'init-enabled'), '?'), 
+                ('uses', '*'),
+            ]
+        ),
+    ),
+    Ext('action', 'ARI'),
+    Ext('condition', 'ARI'),
+    Ext('period', 'ARI'),
+    Ext('start', 'ARI'),
+    Ext('min-interval', 'ARI'),
+    Ext('init-enabled', 'ARI'),
+    Ext('max-count', 'ARI'),
```

Comment:
For these three, can we use native integer / boolean types instead of ARIs?

Reply:
For `min-interval` the value is a `TD` so is an integer appropriate?